### ISSUE TITLE
CONFIG-=import_qpa_plugin is deprecated

### ISSUE THREAD

**Issue by sagitter** [markummitchell/engauge6#10]
Hi all,

this warning appears during ```qmake-qt5``` configuration on ```Fedora 24``` (development branch) and ```Fedora 22``` (stable branch) with ```Qt-5.5.0``` :

```Project WARNING: CONFIG-=import_qpa_plugin is deprecated. Use QTPLUGIN.platforms=- instead.```

Full log: https://kojipkgs.fedoraproject.org//work/tasks/7634/11227634/build.log

**Comment by markummitchell** [comment]
This warning is an artifact from include 'qt' as one of the CONFIG options. I have removed that option, and the problem seems to have disappeared. Tested in Fedora 22. The problem was not appearing in Kubuntu.

**Comment by markummitchell** [comment]
Unfortunately, I realized that without the 'qt' option, developers have to manually define the individual INCLUDEPATH and LIB entries for the QtCore, QtHelp, QtNetwork, QtWidgets, QtXml packages. This causes problems, including (1) the new INCLUDEPATH and LIB entries are absolute paths that change from computer to computer, and (2) having computer-specific information in the project file conflicts with the goals of Qt project files which are that they are simple and universal. 

I restored the 'qt' option.

At this point, there seem to be two options: (1) wait for somebody to fix qmake on systems like Fedora which put out that warning and (2) put in a 'grep -v "import_qpa_plugin"' hack in build scripts.

Suggestions?
